mintapi/api.py

```diff
@@ -789,29 +799,22 @@ def get_transactions_json(self, include_investment=False,
             txns = data['set'][0].get('data', [])
             if not txns:
                 break
-            if start_date:
```

Comment:
Is the idea here that we can use startDate and endDate directly in the calls above, and don't need to filter ourselves anymore?

Reply:
That's correct.  I was running into some friction with the fact that we were using the filtering to also break out of the loop.  So I couldn't exactly do the same thing that we did with start_date.  I explored a little more of the methods and saw that there was another instance of using startDate and endDate.  I then went ahead and tested out the behavior myself and got the desired results.